cmake/modules/AddSwiftUnittests.cmake

```diff
@@ -10,6 +10,9 @@ function(add_swift_unittest test_dirname)
   # function defined by AddLLVM.cmake.
   add_unittest(SwiftUnitTests ${test_dirname} ${ARGN})
 
+  target_include_directories(${test_dirname} PUBLIC ${LLVM_MAIN_SRC_DIR}/utils/unittest/googletest/include)
+  target_include_directories(${test_dirname} PUBLIC ${LLVM_MAIN_SRC_DIR}/utils/unittest/googlemock/include)
```

Comment:
`PUBLIC` is definitely not the right thing here.  The unit tests do not define the interfaces for gtest or gmock.